docs/syntax/flowchart.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Flowcharts Syntax
+outline: 'deep' # shows all h3 headings in outline in Vitepress
+---
+
```

Comment:
If the change is intended only for vitepress, shouldn't we use the `vitepress` flag to filter it out from the regular docs?

Reply:
That makes sense! Fixed in https://github.com/mermaid-js/mermaid/pull/3962/commits/816f2f512e4ff85d9ce9be8b25f0686815640c23